build/release/latest_release_branch.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# Get list of release- branches
+RELEASE_BRANCHES="$(git branch -r | grep -v mergify | grep '/release-' | cut -d'/' -f2 | sort | uniq)"
+LATEST_RELEASE_BRANCH=""
+MAX_TRIES=5
+
+TRY=1
+until [ -n "${LATEST_RELEASE_BRANCH}" ] || [ ${TRY} -ge ${MAX_TRIES} ]; do
```

Comment:
```suggestion
until [ -z "${LATEST_RELEASE_BRANCH}" ] || [ ${TRY} -ge ${MAX_TRIES} ]; do
```
`-n` indicates string is `not` empty and I see `LATEST_RELEASE_BRANCH=""` am I missing something?

Reply:
`until` runs until the condition is true, so the check is running till either the `LATEST_RELEASE_BRANCH` is not empty (`-n`) anymore or the max tries is reached.

The `LATEST_RELEASE_BRANCH` is reset when the release/ branch "is not a valid release branch" (e.g., only alpha/beta tags).

Reply:
ahh I missed  until. Looks good then. Thanks!